src/lib/TranscriptEditor/TimedTextEditor/WrapperBlock.module.css

```diff
@@ -1,44 +1,96 @@
 @value color-labs-red, color-light-grey, color-mid-grey, color-dark-grey from '../colours.module.css';
 
-.WrapperBlock {
-  height: 100%;
-  width: 100%;
-  margin-bottom: 2em;
-  line-height: 1.2em;
-  overflow: hidden;
-}
+/* Desktop size */
+@media (min-width: 768px) {
 
-.markers {
-  width: 25%;
-  font-size: 0.8em;
-  line-height: 1.2em;
-  float: left;
-}
+  .markers {
+    width: 25%;
+    font-size: 0.8em;
+    line-height: 1.2em;
+    float: left;
+  }
+
+  .WrapperBlock {
+    height: 100%;
+    width: 100%;
+    margin-bottom: 2em;
+    line-height: 1.2em;
+    overflow: hidden;
+  }
+
+  .speaker {
+    display: inline-block;
+    width: 50%;
+    padding-right: 2em;
```

Comment:
This offsets everything onto a new line:
<img width="549" alt="screen shot 2019-02-21 at 14 27 45" src="https://user-images.githubusercontent.com/9170575/53175957-fe7e0c80-35e4-11e9-885c-a9c36b2acabb.png">

Reply:
This was quite tricky to get right the first time (even though it wasn't perfect), and so i'm reluctant to merge in these changes. In addition to the iphone / ipad view, the middle-sizes on desktop should be checked too.

There's a lot of space around the video player, where it could either take up the `aside` width _or_ give more room to the editor

<img width="849" alt="screen shot 2019-02-21 at 14 29 30" src="https://user-images.githubusercontent.com/9170575/53176036-24a3ac80-35e5-11e9-8809-a926a8ff4072.png">

Reply:
Yes, I think these needs to be addressed when we look at the media query breakpoints for the responsive design. So that as you said we can tweak for each size the various un-utilised spaces etc..